Firebase/Storage/FIRStorageMetadata.m

```diff
@@ -180,6 +180,10 @@ - (NSDictionary *)dictionaryRepresentation {
   if (_path) {
     metadataDictionary[kFIRStorageMetadataName] = _path;
   }
+  
+  if (_size) {
+    metadataDictionary[kFIRStorageMetadataSize] = [NSNumber numberWithLong:_size];
```

Comment:
I addressed this warning and it no longer shows up in Travis. It now says that the Pod passed validation. I can't reproduce the original warning in XCode though (long is 64 bits on 64 bit machines, so it should have been sufficient).